cyences_app_for_splunk/default/macros.conf

```diff
@@ -386,6 +386,9 @@ definition = 0.5
 # no. of packets in millions
 definition = 0.5
 
+[cs_network_outbound_min_MB_traffic]
+definition = 10
```

Comment:
Not sure on this value

Reply:
As outbound traffic varies for each src_ip maybe we can keep the minimal value 1 MB to avoid the false positives.

Reply:
The problem is what if someone sending empty packets to attack the firewall somehow?

Reply:
yes, correct but this is outbound traffic so I guess this is not a case of attacker, right?

actually upperbound value becomes 0 due to very low traffic. Can we use the max value of the traffic instead of avg. traffic only when avg. traffic is 0? this way we can see some valid upperbound instead 0.